setup.py

```diff
@@ -1,27 +1,19 @@
 #!/usr/bin/env python3
-
 from setuptools import setup, find_packages
-
 import pathlib
-
 HERE = pathlib.Path(__file__).parent
-
-
 def read(path):
     return (HERE / path).read_text("utf-8").strip()
-
-
 install_requires = [
     'aiohttp>=3.5.4,<4.0.0',
     'aiofiles>=0.0.4',
-    'pymongo<=3.11.2',
+    'pymongo>=3.8.1',
     'marshmallow<=3.10.0',  # We have temporary backwards compatibility for 2.X, but also support 3.X
     'cryptography>=2.8',
-    'colorama<=0.4.4',
-    'click<=7.2',
+    'colorama>=0.4',
+    'click>=7.2',
```

Comment:
I don't think we need a version range for click actually, so simply `click` would be sufficient.